bloom-housing/bloom
# Feat/partners breadcrumbs nav

# Pull Request Template

## Issue Overview

This PR addresses #1834


## Description

- removes a back button from listing-related and application-related pages (list, view single, add, edit)
- adds breadcrumbs (for pages mentioned above)
- adds sub-nav for single listing and listing applications page

## How Can This Be Tested/Reviewed?

1. Open partners site
2. Open a listing in draft state `/listings/xxx` - you should see just breadcrumbs
3. Open a listing in the published state - you should see breadcrumbs and sub-nav
4. Click on the "applications" tab - you should land on an application page and see sub-nav
5. Repeat (4) for `Flags` (make sure that you have `SHOW_DUPLICATES=TRUE` in your `.env` file)
6. Click through all listing pages (single listing, edit, add) - you should see proper breadcrumbs
7. Click through all application pages (single application, edit, add)  - you should see proper breadcrumbs

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
